### DISCUSSION TITLE
Azure CLi missing from new CodeSpaces

### DISCUSSION THREAD

**Discussion by bbenz** [community/community#78887] [Codespaces]
### Select Topic Area

Bug

### Body

Hi there!  The azure CLI (AZ) and Azure Dev CLi (AZD) do not appear to be installed in the bash terminal of new codespaces.  I recall that they used to be part of the default CodeSpace image without any extensions or installs needed, and that was really convenient, wondering if this is intentional or did somebody miss something in the latest image?

**Comment by crazy4pi314** [comment]
Did the repos your were working in have devcontainers? The generic, [default Codespace image](https://github.com/devcontainers/images/tree/main/src/universal) afaik has never had those CLI. Sometimes I belive the Codespaces service will try to intelegently pick a more appropriate [prebuilt Dev Containers](https://containers.dev/templates) based on what is in the repo, so maybe it was auto-picking one with those CLIs installed?
